pub-matic-htb.js

```diff
@@ -65,15 +66,154 @@ function PubMaticHtb(configs) {
      * @private {object}
      */
     var __profile;
+     /**
+     * Instances of BidTransformer for transforming bids.
+     *
+     * @private {object}
+     */
+    var __bidTransformers;
```

Comment:
Hi, 

The __bidTransformers are completely removed from the new template updated on master, can you please check against the master branch and modify the code accordingly?

Thanks